examples/python/README.md

```diff
@@ -0,0 +1,27 @@
+## Set up pip
+### Insert the following snippet into your [.pypirc](https://packaging.python.org/en/latest/specifications/pypirc/) (~/.pypirc)
+```pypi
+[distutils]
+index-servers =
+vml-apis-python
+
+[vml-apis-python]
+repository: https://europe-west4-python.pkg.dev/dev-vml-cm/vml-apis-python/
+
+```
+
+### Insert the following snippet into your [pip.conf](https://pip.pypa.io/en/stable/cli/pip_config/)
+
+```
+[global]
+extra-index-url = https://europe-west4-python.pkg.dev/dev-vml-cm/vml-apis-python/simple/
+```
+or run this command
+```bash
+python -m pip config set global.extra-index-url https://europe-west4-python.pkg.dev/dev-vml-cm/vml-apis-python/simple/
+```
+
+## Import grpcio and vml-proto package
+```
+pip install grpcio vml-proto
```

Comment:
Since you're also leaving the `requirements.txt` file in the repo (which is very nice), I's probably better to do `pip install -r requirements.txt` here.

Reply:
Do we want instructions for adopting the sdks, or do we want instructions to run the examples ?

Reply:
I see! Fair point, I got a bit lost in thoughts 😄 You're right, this is good, we want instructions for adopting the SDKs.